src/js/populate_question_form.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+/* global wordle_globals */
+
+var inverted_wordles = {};
+
+// Populate the data on the answer question page
+inverted_wordles.setFormData = function (response, options) {
+    response.json().then(questionFile => {
+        inverted_wordles.setEscapedContent(options.selectors.questionTitle, questionFile.title);
```

Comment:
Actually I think the utility function is unnecessary - you can just use the node's textContent property - https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent

Reply:
A super handy feature I didn't know. 👍